x-pack/plugins/ml/public/application/components/chart_tooltip/chart_tooltip.tsx

```diff
@@ -9,7 +9,9 @@ import type { FC } from 'react';
 import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
 import classNames from 'classnames';
 import TooltipTrigger from 'react-popper-tooltip';
-import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, useEuiMemoizedStyles } from '@elastic/eui';
+// @ts-ignore Kibana does not have types for eui/lib
```

Comment:
```suggestion
// @ts-expect-error Kibana does not have types for eui/lib
```

@ts-expect-error is better here as it'll catch when these types are fixed.